python/lib/database_lib/visit_windows.py

```diff
@@ -44,6 +44,5 @@ def check_visit_label_exits(self, visit_label):
         """
 
         query = 'SELECT Visit_label FROM Visit_Windows WHERE BINARY Visit_label = %s'
-        results = self.db.pselect(query=query, args=(visit_label,))
-
-        return results if results else None
+        results = self.db.pselect(query=query, args=[visit_label])
+        return bool(results)
```

Comment:
Have you checked that changing the return value type does not break any other code?

Reply:
Yep ! The function only has one reference in the codebase (the place I changed).